app/views/NewServerView/hooks/useServersHistory.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Q } from '@nozbe/watermelondb';
+
+import { sanitizeLikeString } from '../../../lib/database/utils';
+import { TServerHistoryModel } from '../../../definitions';
+import database from '../../../lib/database';
+
+const useServersHistory = () => {
+	const [serversHistory, setServersHistory] = useState<TServerHistoryModel[]>([]);
+
+	const queryServerHistory = async (text?: string) => {
```

Comment:
Data comes from local db, but it's still a request done on `onChangeText`.
Should we add a `useDebounce` here?
I'm aware we didn't have it before, but can you give it a try quickly?
I think that's probably a best practice 😬